crates/topos-api/src/conversions/shared/v1/certificate.rs

```diff
@@ -2,7 +2,7 @@ use super::v1::CertificateId;
 
 impl std::fmt::Display for CertificateId {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", hex::encode(&self.value))
+        write!(f, "0x{}", hex::encode(&self.value))
```

Comment:
We can switch to binary `base64` but then debug printout is not usable from the context of our APP, because GRPC encoding/decoding is transparent to us (unless we are debugging/fixing GRPC protocol itself).

Reply:
Ok, lets then switch to base64 for Display of protobuf generated structures